auth-service/src/main/java/com/petclinic/authservice/businesslayer/UserService.java

```diff
@@ -35,6 +35,7 @@
 
 public interface UserService {
 
+    List<UserDetails> getAllUsers();
```

Comment:
I'm confused as to why you put a List here. Christine mentioned that it didn't need to be reactive, so using lists is fine, But then why did you switch to using flux in tthe UserController and the AuthServiceClient ?

Reply:
ya basically since auth service is non reactive we have to put allusers in a list but by using bodyToFlux(UserDetails.class)
we can turn that list back into a reactive stream